### ISSUE TITLE
Command that takes the current word or selection and sends it as an arg

### ISSUE THREAD

**Issue by andycraig** [Ikuyadeu/vscode-R#201]
Another idea I had is a variant of this that take the current word or selection and sends it as an arg.

Would enable stuff like:

```
[
  {
    "description": "loadd target under cursor",
    "key": "ctrl+l",
    "command": "r.somethingSomethingWithSelectionOrWord",
    "when": "editorTextFocus && editorLangId == 'r'",
    "args": "drake::loadd"
  }
```

Thoughts on that one?

_Originally posted by @MilesMcBain in https://github.com/Ikuyadeu/vscode-R/issues/199#issuecomment-577945409_

**Comment by andycraig** [comment]
There's a function `runSelectionOrWord` that if called with `runSelectionOrWord(["t", "head"]))` will send `head(t(SELECTION))` to the console. It sounds like that would do the job if it were exposed in `package.json`.

I've been thinking of changing it so you'd call it like with 'before selection' and 'after selection' arguments: `runSelectionOrWord("head(t(", "))")`. That would enable more control, such as quoting the selection or passing it to a named parameter. Knitting the selection could then be done as discussed here: https://github.com/Ikuyadeu/vscode-R/issues/155#issuecomment-573288608

**Comment by andycraig** [comment]
Closed by #237. Thanks again @MilesMcBain!
